docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to provision a cluster, and to deploy and update Kyma with alpha commands.
+
+## Provision a cluster
+To provision a kubernetes cluster, run:
+
+```bash
+kyma alpha provision k3s 
+```
+
+
+## Install Kyma
+
+There are several ways to install Kyma:
+
+- To install Kyma on a local cluster, you can simply use the `deploy` command.<br>
```

Comment:
Normally these HTML tags like `<br>` will not really work except on GitHub, so we don't use them... Also it seems like on the [deployment preview](https://github.com/kyma-project/cli/pull/764#issuecomment-781419136) the code block is not formatted correctly, I'm not sure if it has anything to do with the HTML 🤔 
<img width="1014" alt="Screen Shot 2021-02-22 at 08 32 33" src="https://user-images.githubusercontent.com/49867570/108676608-ca49c900-74e8-11eb-9fa5-a8a19826402e.png">
```suggestion
- To install Kyma on a local cluster, you can simply use the `deploy` command.
```